deploy/charts/rook-ceph/values.yaml

```
@@ -269,6 +304,7 @@ csi:
   # pluginNodeAffinity: key1=value1,value2; key2=value3
   #cephfsGrpcMetricsPort: 9091
   #cephfsLivenessMetricsPort: 9081
+  #cephnfsLivenessMetricsPort: 9082
```

Is driver a needed distinction?

```suggestion
  #cephNfsDriverLivenessMetricsPort: 9082
```